frontend/src/routes/Governance/policies/Policies.tsx

```diff
@@ -240,9 +243,10 @@ export default function PoliciesPage() {
                     {
                         variant: 'bulk-action',
                         id: 'enable',
+                        // TODO RBAC
+                        // isDisabled: (...item),
```

Comment:
this still a TODO?

Reply:
No, I think this will be handled by the operation when called.